technical/GITHUB_ACTIONS.md

```diff
@@ -20,40 +20,42 @@ Github Actions are configured using YAML files. These files are stored in the `.
 
 ### Building
 All source code repositories need some kind of building step.
-By default, all source code repositories use Gradle as the build tool.
+By default, all source code repositories use Maven as the build tool.
 
-This building step is pretty easy to configure. Just create a `gradle_build.yml` file in the `.github/workflows` directory containing the following source code:
+This building step is pretty easy to configure. Just create a `maven_build.yml` file in the `.github/workflows` directory containing the following source code:
 
 ```yaml
-name: Gradle Build
+name: Maven Build
 
 on: push #(1)
 
 jobs:
   build:
-
+    name: Build
     runs-on: ubuntu-latest
+    timeout-minutes: 15
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 1.11
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.11
-    - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
-    - name: Build with Gradle
-      run: ./gradlew clean build #(2)
-      env: 
-        GITHUB_USERNAME: "OWNER" #(3)
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} #(4)
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+      - name: Create custom Maven Settings.xml #(2)
+        uses: whelk-io/maven-settings-xml-action@v18
+        with:
+          output_file: custom_maven_settings.xml
+          servers: '[{ "id": "github-packages-compas", "username": "OWNER", "password": "${{ secrets.GITHUB_TOKEN }}" }]'
+      - name: Build with Maven
+        run: mvn -s custom_maven_settings.xml -B clean verify #(3)
 ```
 
 A few points to remember:
 - (1): By default, all actions are triggered on a push action.
-- (2): This is a default for building a Gradle project. It may differ in some cases, please feel free to adjust it.
-- (3): Only applicable if your repository is depending on our [Github Package](https://github.com/orgs/com-pas/packages). The GITHUB_TOKEN needs to be set with something (value doesn't matter) to give your access to the Github Packages during build.
-- (4): Again, only applicable if your repository is depending on our Github Packages. The GITHUB_TOKEN gives you access to the Github Packages during build.
+- (2): Only applicable if your repository is depending on our Github Packages.
+  This step creates a custom settings.xml for authenticating for Github Packages. For more information,
+  check the [Contributing file](https://github.com/com-pas/contributing/blob/master/CONTRIBUTING.md).
```

Comment:
Maybe always just add this steps, otherwise you also need to explain that the option "-s custom_maven_settings.xml" needs to be removed.

Reply:
Good point!